popSimManu.tex

```diff
@@ -574,13 +601,15 @@ \section*{Discussion}
 % All of the important points are made below (even with some of the same
 % phrases, such as "barriers to entry") and I think it is needlessly
 % redundant.
+% ALG: I'm not sure about that. I don't really see that it's covered below.
```

Comment:
I think it is covered elsewhere; the question is whether we want to emphasize it. I felt like emphasizing it, especially given the work we've put in to the API, but am leaving this commented out.

Reply:
Hmm... I'd have to give it another full read through.